honeybee/radiance/command/rcollate.py

```diff
@@ -0,0 +1,101 @@
+# coding=utf-8
+from commandBase import RadianceCommand
+import os
+import sys
+from ..datatype import *
+from ..datatype import RadInputStringFormatter as Rfmt
+
+
+class Rcollate(RadianceCommand):
+    u"""
+    rcollate - resize or transpose matrix data
+
+    Attributes:
+
```

Comment:
@sariths here I was thinking just to return a multi-line string that shows the usage code.

Back to testing as probably you have seen I just started writing a unittest. We can create a test-room.rad in test folder that we can use for all test cases. That shouldn't be a problem as far as we make sure that the test will run in an acceptable time-frame.